Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/02-Navigation/03-TestingPresentation/02-03-03-code-0010.swift

```diff
@@ -22,8 +22,10 @@ struct ContactsFeatureTests {
     await store.send(.deleteButtonTapped(id: UUID(1))) {
       $0.destination = .alert(.deleteConfirmation(id: UUID(1)))
     }
-    await store.send(.destination(.presented(.alert(.confirmDeletion(id: UUID(1)))))) {
-      $0.contacts.remove(id: UUID(1))
+    await store.send(\.destination.alert.confirmDeletion, UUID(1)) {
+      $0.contacts = [
+        Contact(id: UUID(0), name: "Blob")
+      ]
```

Comment:
📓 I update this to follow the best practice, you mentioned on Counter tests, to assert on real value instead of the algorithm done in production code 🤔